README.md

```diff
@@ -1,2 +1,124 @@
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 Ruby Capstone <a name="about-project"></a>
+*Ruby Capstone* is a Microverse exercise project which is build with Ruby to manage users catalog using CLI. User can add new books, music albums, and games and also display their catalog list and also they can save their catalog inside Json files.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<summary>Back-end</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/">Ruby</a></li>
+  </ul>
+
+### Key Features <a name="key-features"></a>
+
+- **Users can create new books, music albums, and games using this console app**
+- **Users can see list of books, music albums, and games**
+- **Users can see list of labels, genres, and authors**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+This repository includes ruby code that can be use to decode the morse code.
+
+- At first Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/maov19/ruby-capstone.git
+```
+
+- To use this repo to your local computer make sure you have installed [Ruby](https://www.ruby-lang.org/)
+- To install necessary require packages open your favorite cd to this project then try the below command
+```
+gem install
+```
+- To run this project on your local machine `Enter` the below command
+```
+ruby main.rb
+```
+
+- To run this project tests on your local machine you have to install `Rspec` using this command `gem install rspec` then enter the below command on your terminal
+```
+rspec spec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Abu Raihan**
+
+- GitHub: [@raihan2bd](https://github.com/raihan2bd)
+- Twitter: [@raihan2bd](https://twitter.com/raihan2bd)
+- LinkedIn: [raihan2bd](https://linkedin.com/in/raihan2bd)
+
+👤 **Martin Ossa**
+
+- GitHub: [@maov19](https://github.com/maov19)
+- Twitter: [@maov08](https://twitter.com/maov08)
+- LinkedIn: [Martin Ossa](https://linkedin.com/in/martin-ossa)
+
+👤 **Koshao Godfrey Kazungu**
+
+- GitHub: [@gKoshao-sys](https://github.com/Koshao-sys/)
+- Twitter: [@KoshaoKG](https://twitter.com/KoshaoKG)
+- LinkedIn: [koshaokazungu](https://www.linkedin.com/in/koshaokazungu/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **We will add update catalog feature to this project**
+- [ ] **We will add remove catalog feature to this project**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
It would be great if you added the link to the issues page in place of **_../../issues/_**